frontends/p4/fromv1.0/programStructure.cpp

```diff
@@ -476,7 +472,9 @@ void ProgramStructure::createParser() {
     conversionContext.clear();
 }
 
-void ProgramStructure::include(cstring filename) {
+void ProgramStructure::include(cstring filename, cstring ppoptions) {
+    if (included_files.count(filename)) return;
```

Comment:
This isn't used for `#include` -- it's used to pull in (target specific) header files when translating some construct that requires said header file.  For this, it's nice to not have to worry about including the file mulitple times if mulitple such constructs are used.